api/v3/Generic/Setvalue.php

```diff
@@ -57,8 +57,8 @@ function civicrm_api3_generic_setValue($apiRequest) {
   $fields = civicrm_api($entity, 'getFields', [
     'version' => 3,
     'action' => 'create',
-    "sequential"]
-  );
+    'sequential' => 1,
```

Comment:
@eileenmcnaughton seems this was very odd for a while

Reply:
We should be careful here as fixing the incorrect syntax will change the output format by stripping the array keys. And looking farther down it seems to be relying on those array keys.

Reply:
Looks like "fixing" this line caused broken tests. I recommend removing it instead.